app-modules/interaction/graphql/interaction.graphql

```diff
@@ -203,35 +218,42 @@ input CreateInteractionInput {
     division_id: UUID! @rules(apply: ["required", "exists:divisions,id"])
 
     "The start datetime of the interaction."
-    start_datetime: DateTime!
-        @rules(apply: ["required", "date_format:Y-m-d H:i:s"])
+    start_datetime: DateTime! @rules(apply: ["required", "date:Y-m-d H:i:s"])
```

Comment:
Looks like there are still a few places that we are calling `date` here. Let's go through and make sure we clear up all instances of that.